src/pages/mesh/advanced/extend/resolvers/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Declarative resolvers
+description: Learn how to extend the unified schema with configuration-based resolvers.
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Declarative resolvers
```

Comment:
I'm assuming this topic mostly originates from The Guild. There's a lot of future tense in this topic, but I'm not editing the instances based on that assumption